docs/multihost-networking-tutorial.md

```diff
@@ -0,0 +1,254 @@
+    user@master:~$ ldocker run swarm join --addr=192.168.196.18:2375 token://`cat swarm_id`
+    user@master:~$ ldocker run swarm join --addr=192.168.196.19:2375 token://`cat swarm_id`
+
+std[in|out] will be busy, these commands need to be ran on different terminals. Adding `-d` before the `join` should solve this and enables a for-loop to be used for the joins.
+
+To check if the joins has succeed, run the command:
+
+    user@master:~$ ldocker ps
+    CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                    NAMES
+    d1de6e4ee3fc        swarm               "/swarm join --addr=1"   5 seconds ago       Up 4 seconds        2375/tcp                 fervent_lichterman
+    338572b87ce9        swarm               "/swarm join --addr=1"   6 seconds ago       Up 4 seconds        2375/tcp                 mad_ramanujan
+    7083e4d6c7ea        swarm               "/swarm join --addr=1"   7 seconds ago       Up 5 seconds        2375/tcp                 naughty_sammet
+    0c5abc6075da        swarm               "/swarm join --addr=1"   8 seconds ago       Up 6 seconds        2375/tcp                 gloomy_cray
+    ab746399f106        swarm               "/swarm manage token:"   25 seconds ago      Up 23 seconds       0.0.0.0:5732->2375/tcp   swarm-master
+
+### After the discovery of the swarm members
+To check if the Docker cluster is healty and running, run the command:
+
+    user@master:~$ swarm-docker info
+    Containers: 4
+    Images: 4
+    Role: primary
+    Strategy: spread
+    Filters: health, port, dependency, affinity, constraint
+    Nodes: 4
+     bugs16: 192.168.196.16:2375
+      └ Containers: 0
+      └ Reserved CPUs: 0 / 12
+      └ Reserved Memory: 0 B / 49.62 GiB
+      └ Labels: executiondriver=native-0.2, kernelversion=3.16.0-4-amd64, operatingsystem=Debian GNU/Linux 8 (jessie), storagedriver=aufs
+     bugs17: 192.168.196.17:2375
+      └ Containers: 0
+      └ Reserved CPUs: 0 / 12
+      └ Reserved Memory: 0 B / 49.62 GiB
+      └ Labels: executiondriver=native-0.2, kernelversion=3.16.0-4-amd64, operatingsystem=Debian GNU/Linux 8 (jessie), storagedriver=aufs
+     bugs18: 192.168.196.18:2375
+      └ Containers: 0
+      └ Reserved CPUs: 0 / 12
+      └ Reserved Memory: 0 B / 49.62 GiB
+      └ Labels: executiondriver=native-0.2, kernelversion=3.16.0-4-amd64, operatingsystem=Debian GNU/Linux 8 (jessie), storagedriver=aufs
+     bugs19: 192.168.196.19:2375
+      └ Containers: 4
+      └ Reserved CPUs: 0 / 12
+      └ Reserved Memory: 0 B / 49.62 GiB
+      └ Labels: executiondriver=native-0.2, kernelversion=3.16.0-4-amd64, operatingsystem=Debian GNU/Linux 8 (jessie), storagedriver=aufs
+    CPUs: 48
+    Total Memory: 198.5 GiB
+    Name: ab746399f106
+
+
+At this point swarm is deployed and all containers run will be run over different nodes. Sebian containers can be deploy with the command:
```

Comment:
Typo. At this point swarm is deployed and all containers will be running over different nodes. Sebian containers can be deployed with the command: